python/pip_package/setup.py

```diff
@@ -0,0 +1,103 @@
+# pylint: disable=invalid-name, exec-used
+"""Setup TVM package."""
+from __future__ import absolute_import
+import os
+import shutil
+import sys
+import platform
```

Comment:
is it possible to merge this with python/setup.py? seems a lot of duplication here. Maybe consider have a python file that can get imported